public_html/templates/index.html

```diff
@@ -71,7 +71,7 @@
 					<div class="row-div col-sm-3 text-center wikiproject-div">
 						{% if row.wikiprojects is not empty %}
 							{% for wp in row.wikiprojects %}
-								<span class="row-div wproject">{{ wp }}</span>
+								<a class="row-div wproject" href="#">{{ wp }}</a>
```

Comment:
Changing on hover won't work since we're trying to improve readability, people shouldn't have to hover for that. I leave the decision for this up to you. :)